internal/pkg/featureflags/feature_flags.go

```diff
@@ -57,6 +61,8 @@ type launchDarklyManager struct {
 	isDisabled               bool
 	timeoutSuggestionPrinted bool
 	version                  *version.Version
+	command                  *cobra.Command
+	flags                    []string
```

Comment:
sort the fields? I think things like command and flags should come first than `version`